framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -27,7 +27,11 @@
  *   <li>add the {@code javac.jar} to the runtime classpath of the process that runs the Checker
  *       Framework.
  *   <li>add {@code jdk8.jar} to the compile time bootclasspath of the javac argument list passed to
- *       javac
+ *       javac while using Java 8
+ *   <li>patch JDK modules while using Java9+ using {@code --patch-module
+ *       &lt;module&gt;=&lt;annotatedModule&gt;} annotated modules are present in {@code
+ *       checker/dist/annotatedJDK/jdk*\/} directory. All {@code --patch-module} arguments are
```

Comment:
Say what the programmer should do.  This point starts saying to use `--patch-module`, but then says one should *not* use `--patch-module`, which is confusing.  Don't include implementation details, which is everything about `--patch-module`.

Reply:
Done